include/jemalloc/internal/tsd_internals.h

```diff
@@ -96,7 +98,9 @@ typedef ql_elm(tsd_t) tsd_link_t;
 #define TSD_DATA_SLOW_INITIALIZER					\
     /* tcache_enabled */	TCACHE_ENABLED_ZERO_INITIALIZER,	\
     /* reentrancy_level */	0,					\
+    /* thread_tcache_max */	TCACHE_MAXCLASS_LIMIT + 1,		\
     /* min_init_state_nfetched */	0,				\
+    /* thread_nhbins */		TCACHE_NBINS_MAX + 1,			\
```

Comment:
Why +1 for the two default values?

Reply:
By initializing them to invalid values in the initializer, they will always be properly set [here](https://github.com/jemalloc/jemalloc/pull/2493/files#diff-6e884ea618aef238f8b7415ae1e2e0dad152df27bd06df1c4e366cef1469efdeR716). Another choice is that we can set it to another valid const values, but will have to add a flag to tsd_tcache_data_init so it does not get reset during the reboot.